CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to AET will be documented in this file.
 ## Unreleased
 **List of changes that are finished but not yet released in any final version.**
 
+- [PR-296](https://github.com/Cognifide/aet/pull/296) The behavior of StatusCodesComparator was changed
```

Comment:
Could you please add more information here?
E.g. Status Code Comparator check now range `400-600` by default, parameters validation added